pyEpiabm/pyEpiabm/utility/covidsim_kernel.py

```diff
@@ -0,0 +1,31 @@
+#
+# Class of Covidsim-style gravity kernels
+#
+
+
+class Kernel:
```

Comment:
I think this is only a spatial Kernel right? Might be worth calling it `SpatialKernel` to avoid confusion